src/defaultSettings.js

```diff
@@ -3,15 +3,17 @@ import clone from './util/clone';
 export default {
   //custom settings
   form_col: 'form',
+  formDescription_col: null,
```

Comment:
I think leaving this null make sense, but we'll likely want to make a second example with `settings` as follows:
```
settings = {
      formDescription_col:"formDescription",
      fieldDescription_col:"fieldDescription"
  }
```